docs/rules/no-promise-without-done-fail.md

```diff
@@ -0,0 +1,37 @@
+# Disallow using promise without a done.fail (no-promise-without-done-fail).
+
+Make sure that all uses of the promises include a catch(done.fail) within a suite.
+
+## Rule details
+
+The following are considered warnings:
+
+```js
+describe('A suite', function(done) {
+  it('A spec', function() {
+    asyncCall().then(done);
+  });
+});
+
+describe('A suite', function(done) {
+  it('A spec', function() {
+    asyncCall().then(function() {
+      expect(true).toBe(true);
+      done();
+    });
+  });
+});
+```
+
+The following patterns are not warnings:
+
+```js
+describe('A suite', function(done) {
+  it('A spec', function() {
+    asyncCall().then(function() {
+      expect(true).toBe(true);
+      done();
+    }).catch(done.fail);
```

Comment:
Is  `.then(onSuccess, done.fail)`considered to be correct? if so can you add this to the doc

Reply:
@DianaSuvorova updated the docs